src/style.css

```diff
@@ -0,0 +1,72 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf) format("truetype");
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(./fonts/Avenir-Heavy.ttf) format("truetype");
+}
+
+.search__input {
+  width: 100%;
+  text-align: left;
+  font-family: Avenir, sans-serif;
```

Comment:
move to `html` selector

Reply:
Moved, except width. It shrinks to size of input and fails... Mb it`s ok?

Reply:
I meant only one property)